Add operational procedure for intermittent API 503 errors (#1599)
Cray-HPE/docs-csm: operations/kubernetes/Troubleshoot_Intermittent_503s.md

# Troubleshoot Intermittent HTTP 503 Code Failures

There are cases where API calls or `cray` command invocations will intermittently return an HTTP 503 code, even when the backing service is up and functional.
In the event that this occurs, the following steps may be useful to remediate the issue.

1. Inspect the `istio-ingressgateway` pod logs:

   ```bash
   ncn-m # kubectl -n istio-system logs -l app=istio-ingressgateway
   ```

>    If you see TLS errors similar to the following proceed to the next step to restart the `istio-ingressgateway` pods.

Reviewer: ```suggestion
   If the logs include TLS errors similar to the following, then proceed to the next step to restart the `istio-ingressgateway` pods.
```
